app.py

```diff
@@ -163,5 +165,37 @@ def setGridBoundries():
                               mimetype='application/json')
 
 
+@app.route('/export-data', methods=['POST'])
```

Comment:
try using non-hyphenated urls eg: `export_data`

Reply:
I believe it is standard practice to use hyphenated urls instead of underscores for API endpoints

Reply:
Have removed this endpoint and shifted the request creating to front-end